### DISCUSSION TITLE
Failed to create sample statisticsCluster has no available capacity

### DISCUSSION THREAD

**Discussion by impremkumar** [StarRocks/starrocks#36204] [How-tos, Tutorials, Best Practices and Architecture Articles]
Team,

Can someone please explain the root cause for these warnings ? 

fe.warn.log:2023-11-30 17:27:20,092 WARN (statistics meta manager|24) [StatisticsMetaManager.createSampleStatisticsTable():143] Failed to create sample statisticsCluster has no available capacity
fe.warn.log:2023-11-30 17:27:20,092 WARN (statistics meta manager|24) [StatisticsMetaManager.refreshStatisticsTable():290] create statistics table table_statistic_v1 failed

Also, I could see both frontend and backend are running without any issues, but I don't see any backend listed under "known backends" section. 

output for "fe" status query - 
 cat fe/log/fe.log | grep thrift
 
 127.0.0.1_9010_1701386012017(-1)|1) [FeServer.start():52] thrift server started with port 9020.

Output for "be" status query

cat be/log/be.INFO | grep heartbeat

 12594 thrift_server.cpp:378] heartbeat has started listening port on 9050

![image](https://github.com/StarRocks/starrocks/assets/34817027/62540024-ec6f-4d55-90ca-26e9b206684f)

Any insights on this issue would be greatly appreciated ?

**Comment by chaoyli** [comment]
You need to add the backend. It need the backend storage.

**Comment by alberttwong** [comment]
thi sisn't hte place to ask questions.  Please use slack.
